### ISSUE TITLE
suggested updates adjusted to Fram

### ISSUE THREAD

**Issue by franzihe** [UNINETTSigma2/documentation#7]
File:  documentation/software/userinstallsw/python.md 
### it is suggested to use: `use 'module list Python' to see all`
- using `module list Python` on Fram leads to 
```
$ module list Python

Currently Loaded Modules Matching: Python
  None found.

``` 
Perhaps this needs to be updated in the documentation to only be available on Saga.

### it is suggested: 
```
$ ml purge
$ ml avail conda
$ ml Miniconda3/4.6.14
```
- using `ml avail conda` on Fram leads to 
```
$ ml avail conda

--------------------------------------------------------------------- /cluster/modulefiles/all ---------------------------------------------------------------------
   Anaconda2/2018.12    Anaconda2/2019.07    Anaconda3/2019.07

Use "module spider" to find all possible modules.
Use "module keyword key1 key2 ..." to search for all possible modules matching any of the "keys".


```
Hence, Miniconda3 is not installed on Fram, but on Saga.

**Comment by bast** [comment]
Thanks for reporting!

**Comment by bast** [comment]
Working on fixing this ...

**Comment by bast** [comment]
I have fixed this in dfaa3707d468b02. Thanks again for reporting.
